spec/support/helpers.rb

```diff
@@ -0,0 +1,9 @@
+module Spec
+  module Support
+    module Helpers
+      def generate_signature(webhook_body)
```

Comment:
I think the key to use should be passed in here, otherwise it's not clear in the code which key will be used.